apps/cli/src/commands/doctor.ts

```diff
@@ -29,6 +29,9 @@ export default class DoctorCommand extends Command {
 
     public async run() {
         try {
+            // Check if Docker is installed
+            await execa("docker", ["--version"]);
+
             // Check Docker version
             const { stdout: dockerVersionOutput } = await execa("docker", [
```

Comment:
`docker` is being called two times here.

We could call it once and use its output to check the version, catch an error and see `error.exitCode` to treat it correctly. `127 exitCode` would be a command not found, for example.